next.config.js

```diff
@@ -1,9 +1,18 @@
+const withPlugins = require('next-compose-plugins')
 const withSvgr = require('next-svgr')
-const withMDX = require('@next/mdx')({
-  extension: /\.mdx?$/,
-})
-module.exports = withSvgr(
-  withMDX({
-    pageExtensions: ['js', 'jsx', 'md', 'mdx', 'tsx', 'ts'],
+const withMdxEnhanced = require('next-mdx-enhanced')
+
+module.exports = withPlugins([
+  withSvgr,
+  withMdxEnhanced({
+    layoutPath: 'layouts',
+    defaultLayout: true,
+    fileExtensions: ['mdx'],
+    remarkPlugins: [],
+    rehypePlugins: [],
+    extendFrontMatter: {
```

Comment:
Here's a search for all `next.config.js` files including `withMdxEnhanced` if we wanna safari what others are doing:

https://github.com/search?q=withMdxEnhanced+extension%3A.js+filename%3Anext.config.js&type=Code